edocumentation-transfer-obj/src/main/java/org/dcsa/edocumentation/transferobjects/ShippingInstructionTO.java

```diff
@@ -90,7 +90,11 @@ public record ShippingInstructionTO(
   @Valid
   List<@Valid @NotNull CustomsReferenceTO> customsReferences,
 
-  List<@Valid @NotNull AdvanceManifestFilingEBLTO> advanceManifestFilings
+  List<@Valid @NotNull AdvanceManifestFilingEBLTO> advanceManifestFilings,
+
+  List<@Size(max = 100) @Pattern(regexp = "^\\S+(\\S+\\S+)$") String> requestedCarrierCertificates,
+
+  List<@Size(max = 100) @Pattern(regexp = "^\\S+(\\S+\\S+)$") String> requestedCarrierClauses
```

Comment:
That pattern does not make sense `^\S+(\S+\S+)$` is basically `^\S{3,}$`.  I think you wanted  `^\S+(\s+\S+)*$` (as that is the only pattern I can think of with that "basic form").